### PR TITLE
Refactor project structure

### PR DESCRIPTION
**Pull request by gayashanbc**

### CODE REVIEW

File: io.asgardio.tomcat.saml.agent.sample/pom.xml [hunk 1]
```diff
@@ -22,17 +22,16 @@
...
     </parent>
     <modelVersion>4.0.0</modelVersion>
 
-    <artifactId>SampleApp</artifactId>
+    <artifactId>io.asgardio.tomcat.saml.agent.sample</artifactId>
     <packaging>war</packaging>
 
     <build>
-        <finalName>SampleApp</finalName>
+        <finalName>io.asgardio.tomcat.saml.agent.sample</finalName>
```
Comment (darshanasbg) [inline io.asgardio.tomcat.saml.agent.sample/pom.xml:33]: Shall we have the application name as `sample-app`.

Current name is too long and format doesn't look nice for the URL once app is deployed.

File: io.asgardio.tomcat.saml.agent.sample/src/main/webapp/WEB-INF/classes/META-INF/LICENSE [hunk 2]
```diff
@@ -1,200 +1,202 @@
...
-      or other liability obligations and/or rights consistent with this\n\
-      License. However, in accepting such obligations, You may act only\n\
-      on Your own behalf and on Your sole responsibility, not on behalf\n\
-      of any other Contributor, and only if You agree to indemnify,\n\
-      defend, and hold each Contributor harmless for any liability\n\
-      incurred by, or claims asserted against, such Contributor by reason\n\
-      of your accepting any such warranty or additional liability.\n\
-\n\
-   END OF TERMS AND CONDITIONS\n\
+
```
Comment (darshanasbg) [inline io.asgardio.tomcat.saml.agent.sample/src/main/webapp/WEB-INF/classes/META-INF/LICENSE:1]: Do we need this?

File: io.asgardio.tomcat.saml.agent.sample/src/main/webapp/WEB-INF/classes/META-INF/DEPENDENCIES [hunk 3]
```diff
@@ -0,0 +1,357 @@
+// ------------------------------------------------------------------
```
Comment (darshanasbg) [inline io.asgardio.tomcat.saml.agent.sample/src/main/webapp/WEB-INF/classes/META-INF/DEPENDENCIES:1]: Do we need this?

File: io.asgardio.tomcat.saml.agent.sample/src/main/webapp/WEB-INF/classes/META-INF/NOTICE [hunk 4]
```diff
@@ -0,0 +1,8 @@
+
```
Comment (darshanasbg) [inline io.asgardio.tomcat.saml.agent.sample/src/main/webapp/WEB-INF/classes/META-INF/NOTICE:1]: Do we need this?

File: pom.xml [hunk 5]
```diff
@@ -302,7 +214,7 @@
 
     <properties>
         <identity.framework.version>5.15.0</identity.framework.version>
-        <identity.sso.agent.package.export.version>${project.version}</identity.sso.agent.package.export.version>
+        <asgario.tomcat.saml.agent.version>${project.version}</asgario.tomcat.saml.agent.version>
```
Comment (darshanasbg) [inline pom.xml:217]: Whats the usage of this property?

File: pom.xml [hunk 6]
```diff
@@ -344,14 +256,11 @@
         <commons-collections.wso2.osgi.version.range>[3.2.0,4.0.0)</commons-collections.wso2.osgi.version.range>
         <encoder.wso2.version>1.2.0.wso2v1</encoder.wso2.version>
         <apache.felix.scr.ds.annotations.version>1.2.4</apache.felix.scr.ds.annotations.version>
-        <io.asgardio.java.saml.sdk.version>0.1.0</io.asgardio.java.saml.sdk.version>
+        <io.asgardio.java.saml.sdk.version>0.1.1-SNAPSHOT</io.asgardio.java.saml.sdk.version>
```
Comment (darshanasbg) [inline pom.xml:259]: Don't have snapshot versions of dependencies.

**Review by darshanasbg — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 5 above)
Comment (gayashanbc) [reply pom.xml:217]: We are using in the sample app pom.xml to add tomcat agent as a dependency of the webapp

File: pom.xml (hunk 6 above)
Comment (gayashanbc) [reply pom.xml]: Resolved in https://github.com/asgardio/asgardio-tomcat-saml-agent/pull/2/commits/7bc0a7b510c5137c156f5317497b12e794ef3993

File: io.asgardio.tomcat.saml.agent.sample/src/main/webapp/WEB-INF/classes/META-INF/NOTICE (hunk 4 above)
Comment (gayashanbc) [reply io.asgardio.tomcat.saml.agent.sample/src/main/webapp/WEB-INF/classes/META-INF/NOTICE]: Removed the file in https://github.com/asgardio/asgardio-tomcat-saml-agent/pull/2/commits/7bc0a7b510c5137c156f5317497b12e794ef3993

File: io.asgardio.tomcat.saml.agent.sample/src/main/webapp/WEB-INF/classes/META-INF/DEPENDENCIES (hunk 3 above)
Comment (gayashanbc) [reply io.asgardio.tomcat.saml.agent.sample/src/main/webapp/WEB-INF/classes/META-INF/DEPENDENCIES]: Removed the file in https://github.com/asgardio/asgardio-tomcat-saml-agent/pull/2/commits/7bc0a7b510c5137c156f5317497b12e794ef3993

File: io.asgardio.tomcat.saml.agent.sample/src/main/webapp/WEB-INF/classes/META-INF/LICENSE (hunk 2 above)
Comment (gayashanbc) [reply io.asgardio.tomcat.saml.agent.sample/src/main/webapp/WEB-INF/classes/META-INF/LICENSE]: Removed the file in https://github.com/asgardio/asgardio-tomcat-saml-agent/pull/2/commits/7bc0a7b510c5137c156f5317497b12e794ef3993

File: io.asgardio.tomcat.saml.agent.sample/pom.xml (hunk 1 above)
Comment (gayashanbc) [reply io.asgardio.tomcat.saml.agent.sample/pom.xml]: Resolved in https://github.com/asgardio/asgardio-tomcat-saml-agent/pull/2/commits/7bc0a7b510c5137c156f5317497b12e794ef3993

File: pom.xml (hunk 5 above)
Comment (darshanasbg) [reply pom.xml:217]: Can't we use project version there directly?

Comment (gayashanbc) [reply pom.xml]: Resolved in https://github.com/asgardio/asgardio-tomcat-saml-agent/pull/2/commits/8c6cf400b66a4e63ec7226c85a6884b7e379fae1

**Review by darshanasbg — APPROVED** [review]

**Review by darshanasbg — APPROVED** [review]

**Pull request merged**